package.json

```diff
@@ -1,8 +1,14 @@
 {
   "name": "@c-hive/team-contribution-calendar",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "GitHub-like contribution calendar for the whole team. Supports GitHub and GitLab.",
-  "main": "./lib/index.js",
```

Comment:
@gomorizsolt does this change make sense?

Reply:
I mean changing main to point to `src`. I think this and removing the build comes from when we wanted to release the source without transpiling but that idea got scratched.

Reply:
Good point. The transpiled version exists in the `lib` folder, so I'd say the `main` should be kept without changes(i.e. it should point to `lib` as before).